src/airmobility/object_a.proto

```diff
@@ -66,16 +67,16 @@ message VoxelValueBuilding {
 }
 
 message ObjectRestrictedArea { // 飛行制限空域（飛行制限区域）の値。空間IDを含む。
-    enum RestrictType{
```

Comment:
"Restrict" is not an adjective but a verb.
The name doesn't describe the content.
So I removed.